lib/google_contacts_api/contacts_accessor.rb

```diff
@@ -1,6 +1,6 @@
 # Module that implements a method to get contacts for a user or group
 module GoogleContactsApi
-  module Contacts
+  module ContactsAccessor
```

Comment:
I ended up keeping the layers between the api and user object, and renamed them to contacts_accessor.  I kept it this way because the group class needed to still have access to the get_contacts method, and putting that into the user class would force a group object to store a reference to the user, which didn't make sense.  Since put contact, post contact, and get contacts are all making api calls pertaining to contacts, I kept them together and tried to give the old 'contacts' (and groups) module a name to reflect this.